clarifai/runners/dockerfile_template/Dockerfile.cuda.template

```diff
@@ -7,36 +7,21 @@
 # * Create a non-root user with minimal privileges and use it
 ARG TARGET_PLATFORM=linux/amd64
```

Comment:
should use `TARGETPLATFORM` which gets set by docker instead

`ARG` also need to be after `FROM`

Reply:
it seems to work when it's before. but maybe it's because we're templating it it just "works"?

Reply:
seems like ARG can be before FROM in their docs. I think we need the target platform to be set indendent of where the build is happening so we can build for other platforms.